### DISCUSSION TITLE
[BLOG • medium.com] Continuous Delivery: Apache Superset docker images

### DISCUSSION THREAD

**Discussion by numanijaz** [apache/superset#24792] [Show and tell]
Dear community members,

I wrote the following article on medium.com to briefly go through how Apache Superset is doing continous delivery of its docker images:

https://medium.com/@numanijaz09/continuous-delivery-apache-superset-docker-images-bd892f200c3d

**Comment by sebastianliebscher** [comment]
Hi @NumanIjaz, thank you for sharing your article. Great contribution to the community!

One small thing I noticed: it states to use the 2.1.0 tag, but in parts the Dockerfile on master branch is used.

**Reply by numanijaz** [reply]
Hi @sebastianliebscher, thanks for noticing, I mixed it up there. The mistakes are now corrected and published.
